_events/2023-01-25-htcondor-osg.md

```diff
@@ -21,20 +21,23 @@ header_image: "/images/capital.jpeg"
 <p style="font-size: larger; font-weight: bold;">Don't miss these in-person learning opportunities in beautiful Madison, Wisconsin!</p>
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the long-standing events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Throughput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing. 
```

Comment:
Please find a way to use a spell-checker. “offerring” => “offering”

Also, this sentence is just wordy. I think there is some repetition of combining/joint and events that could be streamlined. How about: “For the first time, **HTCondor Week** and the **OSG All-Hands Meeting** will join together as a single, integrated event from July 10–14 at …”?